app.js

```diff
@@ -1,178 +1,182 @@
-//using express framework
-var express = require('express');
-var fs = require('fs');
-var path = require('path');
-const logger = require('morgan');
-const bodyParser = require('body-parser');
-var Sequelize = require('sequelize');
-const config = require(`${__dirname}/server/config/config.json`)["development"];
-
-var sequelize = new Sequelize(config.database,config.username, config.password,config.options);
-
-//create express application
-var app = express();
-
-//set up handlebars view engine
+// using express framework
+import express from 'express';
+import fs from 'fs';
+import path from 'path';
+import logger from 'morgan';
+import bodyParser from 'body-parser';
+import Sequelize from 'sequelize';
+import routes from './server/routes';
+import Config from './server/config/config.json';
+
+// import fortune from 'lib/fortune.js';
+const config = Config['development'];
+const sequelize = new Sequelize(config.database,config.username, config.password,config.options);
+
+// create express application
+const app = express();
+
+// set up handlebars view engine
 // var handlebars = require('express3-handlebars').create({defaultLayout:'main'});
 
 // app.engine('handlebars',handlebars.engine);
 // app.set('view engine','handlebars');
 
 
-//set server listening port 
-app.set('port',process.env.PORT || 3000);
+// set server listening port 
+app.set('port', 3000);
```

Comment:
Why do you remove `process.env.PORT ||`? Dont you have `.env` file?

Reply:
No, i don't have a `.env` file

Reply:
You need to install `dotenv` package [dotenv](https://www.npmjs.com/package/dotenv )